File: Java/Gate-Four/FunctionTwo.java
public class FunctionTwo {

    public static int countEvens(int[] numbersList) {
        int evenCount = 0;

        for (int number : numbersList) {
            if (number % 2 == 0) {
                evenCount++;
            }
        }

        return evenCount;
    }
    public static int[] squares(int number) {
        int[] squareNumbers = new int[number];

        for (int i = 1; i <= number; i++) {
            squareNumbers[i - 1] = i * i;
        }

        return squareNumbers;
    }
    public static boolean arraysEqual(int[] firstList, int[] secondList) {
        if (firstList.length != secondList.length) {
            return false;
        }

        for (int i = 0; i < firstList.length; i++) {
            if (firstList[i] != secondList[i]) {
                return false;
            }
        }

        return true;
    }
    

 }

File: Python/Gate-Four/function_two.py
def countEvens(numbers_list):
    even_count = 0
    for number in numbers_list:
        if number % 2 == 0:
            even_count += 1
    return even_count


def squares(number):
    square_numbers = []
    for number in range(1, number + 1):
        square_numbers.append(number * number)
    return square_numbers


def arraysEqual(first_list, second_list):
    if len(first_list) != len(second_list):
        return False
    
    for index in range(len(first_list)):
        if first_list[index] != second_list[index]:
            return False
    
    return True


def removeDuplicates(input_list):
    unique_list = []
    for number in input_list:
        if number not in unique_list:
            unique_list.append(number)
    return unique_list
